app/helpers/application_widgets_helper.rb

```diff
@@ -0,0 +1,43 @@
+module ApplicationWidgetsHelper
+  # Create a edit button with icon
+  #
+  # @param options [String, Hash] The url or the url options which will be passed to `link_to`.
+  #
+  # @param html_options [Hash] The options to specify the class and title of the button.
+  # @return [String] The string including embedded HTML which displays the button with a link to
+  #                  path.
+  def edit_button(options, html_options = {})
+    html_options = html_options.dup
+
+    css_class =  html_options[:class] || []
+    css_class |= ['btn']
+    css_class << 'btn-default' unless css_class.find { |e| /btn-/ =~ e }
```

Comment:
have to hardcode the button types. There's button-large and button-medium etc which would match but not dictate a style.